manage.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+import os
+import sys
+from django.core.management import execute_from_command_line
+
+if __name__ == "__main__":
+    os.environ.setdefault(
+        "DJANGO_SETTINGS_MODULE",
+        "recommender.conf.locale.settings"
```

Comment:
The `locale` directory shouldn't contain python. I haven't seen this pattern used anywhere.

```suggestion
        "recommender.dev_settings"
```